pkg/server/grpc/tracee.go

```diff
@@ -83,3 +118,189 @@ func convertDefinitionToProto(d events.Definition) *pb.EventDefinition {
 		Tags:        d.GetSets(),
 	}
 }
+
+func convertTraceeEventToProto(e trace.Event) *pb.Event {
+	process := getProcess(e)
+	container := getContainer(e)
+	k8s := getK8s(e)
+
+	var eventContext *pb.Context
+	if process != nil || container != nil || k8s != nil {
+		eventContext = &pb.Context{
+			Process:   process,
+			Container: container,
+			K8S:       k8s,
+		}
+	}
+
+	event := &pb.Event{
+		Id:   uint32(e.EventID),
+		Name: e.EventName,
+		Policies: &pb.Policies{
+			Matched: e.MatchedPolicies,
+		},
+		Context: eventContext,
+		Threat:  getThreat(e),
+	}
+
+	if e.Timestamp != 0 {
+		event.Timestamp = timestamppb.New(time.Unix(int64(e.Timestamp), 0))
+	}
```

Comment:
does this mean that timestamp field is optional? because it shouldn't be

Reply:
it is not optional, though I wasn't sure if the current event structure always have a timestamp, so I tested to check it before conversion.